### PR TITLE
Feature/translate v1

### PR DESCRIPTION
**Pull request by cristofer-dev**

### CODE REVIEW

File: evento2018.html [hunk 1]
```diff
@@ -15,20 +15,21 @@
...
+      <span class="rojo f-3vh bold">August 18th</span>
       <br>
       <span class="f-6vh bold">2018</span>
       <br>
       <span>La Serena - Chile</span>
     </div>
 
     <div class="col-md-6 col-sm-6 col-xs-6 text-center  m-t-10vh">
-      <h3 class="rojo">Organiza</h3>
+      <h3 class="rojo">Organize</h3>
```
Comment (Xabadu) [inline evento2018.html]: `Organizers`

File: evento2018.html [hunk 2]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
 
   <div id="objetivo" class="row ">
     <div class="col-md-12  text-center ">
-      <h1>Nuestro Objetivo...</h1>
+      <h1>Our objective...</h1>
```
Comment (Xabadu) [inline evento2018.html]: `Our goal...`

File: evento2018.html [hunk 3]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
...
       <p>
         <span class="rojo bold">></span>
-        Somos una Comunidad que cree firmemente que en el interior de cada persona existe un potencial inigualable para crear y afectar
-        positivamente a las personas y al entorno que nos rodea, más aún cuando esa capacidad de crear se suma a la de otros,
-        comprendiendo asi que el todo es más que
-        <b>la suma de las partes</b>
+        We are a Community that firmly believes that within each person there is an unparalleled potential to create and affect positively to the people and the environment that surrounds us, especially when that capacity to create adds to that of others, understanding that the whole is more than <b> the sum of the parts </b>
```
Comment (Xabadu) [inline evento2018.html]: `affect positively the people and the environment that surrounds us`
`sum of its parts`

File: evento2018.html [hunk 4]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
...
       </p>
       <p>
         <span class="rojo bold">></span>
-        Eso nos mueve a hacer
-        <b>cosas</b> que aporten valor a nuestro entorno con lo que mejor sabemos hacer...
-        <b>la programación</b>. Regularmente realizamos actividades que puedes revisar
-        <a href="http://ivdevs.com/evento">aquí</a> y anualmente organizamos una conferencia con múltiples charlas que buscan ser el
-        <b>update tecnológico</b> que permitan fortalecer tanto para desarrolladores locales como a entusiastas de la tecnología.
+        That moves us to do <b>things</b> that add value to our environment with what we do best ... programming. We regularly carry out activities that you can review <a href="http://ivdevs.com/evento">here</a> and annually we organize a conference with multiple talks that seek to be the technological update that allows strengthening both for local developers and technology enthusiasts.
```
Comment (Xabadu) [inline evento2018.html]: `That pushes us to do`
`activities that you can view`
`and we organize a yearly conference`

File: evento2018.html [hunk 5]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
...
       <p>
         <span class="rojo bold">></span>
-        Todas nuestras actividades son
+        All our activities are
         <b>non-profit.</b>
       </p>
       <p>
         <span class="rojo bold">></span>
-        <a href="/conference2017/">Mira como estuvo la conferencia del 2017</a>
+        <a href="/conference2017/">See how the 2017 conference</a>
```
Comment (Xabadu) [inline evento2018.html]: `See our 2017 conference`

File: evento2018.html [hunk 6]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
...
       </p>
     </div>
   </div>
   <!-- row -->
 
   <div id="numeros" class="row ">
     <div class="col-md-12  text-center ">
-      <h1>Nosotros...</h1>
+      <h1>US...</h1>
```
Comment (Xabadu) [inline evento2018.html]: `About Us`

File: evento2018.html [hunk 7]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
...
     </div>
 
     <div class="col-md-4 col-sm-6 col-xs-12 text-center m-t-5vh ">
       <span class="valor">26</span>
       <br>
-      <span class="textos">Meetups realizados</span>
+      <span class="textos">Meetups made</span>
```
Comment (Xabadu) [inline evento2018.html]: `Previous meetups` (o events)

File: evento2018.html [hunk 8]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
...
-        Las bases de una comunidad es el networking, por ello prestamos especial atención al apoyo entre comunidades, apoyamos y
-        nos apoyan en diferentes actividades:
+        The base of a community is networking, therefore we pay special attention to the support among communities, we support and they support us in different activities
       </p>
     </div>
 
     <div class="col-md-6 col-sm-6 col-xs-12 text-center img">
       <img src="/assets/img/partners/logo_noders_white.png" alt="">
-      <span>La comunidad de NodeJS y JS en Lationamérica</span>
+      <span>The community of NodeJS y JS in Latin America</span>
```
Comment (Xabadu) [inline evento2018.html]: `Node.js`
`The Most Electrifying Node.js community in Latin America`

File: evento2018.html [hunk 9]
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
...
     </div>
     <div class="col-md-6 col-sm-6 col-xs-12 text-center img">
       <img src="http://www.programadores.cl/images/proin-logo-min.png" alt="">
-      <span>Comunidad de Programadores e Informáticos de Chile</span>
+      <span>Community of Programmers of Chile</span>
```
Comment (Xabadu) [inline evento2018.html]: `in Chile`

File: evento2018.html [hunk 10]
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
...
 
     <div class="col-md-4">
-      <h3>Pre Evento</h3>
+      <h3>Pre Event</h3>
       <ul>
-        <li>- Menciones en redes sociales.</li>
-        <li>- Campañas de pago en Facebook.</li>
-        <li>- Sitio web.</li>
+        <li>- Social networks.</li>        
+        <li>- webpage.</li>
```
Comment (Xabadu) [inline evento2018.html]: `Website`

File: evento2018.html [hunk 11]
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
...
         <li>- Radio.</li>
+        <li>- Newsletter.</li>
       </ul>
     </div>
     <div class="col-md-4">
-        <h3>Durante el evento</h3>
+        <h3>During Event</h3>
```
Comment (Xabadu) [inline evento2018.html]: `During the event`

File: evento2018.html [hunk 12]
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
...
     <div class="col-md-4">
-        <h3>Post Evento</h3>
+        <h3>Post Event</h3>
         <ul>
-          <li>- Publicación de charlas en videos.</li>
-          <li>- Publicación de fotografías en Redes sociales.</li>
-          <li>- Publicación de agradecimientos oficiales.</li>
-          <li>- Noticia en sitios Online involucrados.</li>
+          <li>- Publishing of the recorded sessions.</li>
+          <li>- Publishing of the photographic in social networks.</li>
```
Comment (Xabadu) [inline evento2018.html]: `photos`
`social media`

File: evento2018.html [hunk 13]
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
...
     </div>
 
     <div class="col-md-4">
-      <h3>Pre Evento</h3>
+      <h3>Pre Event</h3>
       <ul>
-        <li>- Menciones en redes sociales.</li>
-        <li>- Campañas de pago en Facebook.</li>
-        <li>- Sitio web.</li>
+        <li>- Social networks.</li>
```
Comment (Xabadu) [inline evento2018.html]: `Social media`

File: evento2018.html [hunk 14]
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
...
+          <li>- Sponsor logos among published event photos</li>
         </ul>
     </div>
 
   </div>
   <!-- row -->
   <div id="sponsor" class="row ">
     <div class="col-md-12  text-center ">
-      <h1 class="rojo">Se sponsor</h1>
+      <h1 class="rojo">Be sponsor</h1>
```
Comment (Xabadu) [inline evento2018.html]: `Be a sponsor`

File: evento2018.html [hunk 15]
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
...
     </div>
     <div class="row">
       <div class="col-md-offset-1 col-md-5 p-20">
         <div class="row">
         <h3><strong>GOLD</strong></h3>
-        <h4>Desde $200.000</h4>
+        <h4>400 USD</h4>
```
Comment (Xabadu) [inline evento2018.html]: Cuando el dolar estaba a 500 (?)

File: evento2018.html [hunk 16]
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
...
           <ul>
-            <li>- Presentación del sponsor</li>
+            <li>- Logo in website with link to website</li>
```
Comment (Xabadu) [inline evento2018.html]: `Logo in our Website linked to the sponsor's site`

**Review by Xabadu — CHANGES_REQUESTED** [review]

**Comment by cristofer-dev** [comment]
@Xabadu DONE!

**Review by Xabadu — APPROVED** [review]

**Pull request merged**